src/main/java/org/minions/devfund/angela/battleship/BattleshipGame.java

```diff
@@ -0,0 +1,64 @@
+package org.minions.devfund.angela.battleship;
+
+import java.util.Scanner;
+
+/**
+ * Executes Battleship game.
+ */
+public final class BattleshipGame {
+
+    /**
+     * Private constructor.
+     */
+    private BattleshipGame() {
+
+    }
+
+    /**
+     * Executes the game.
+     *
+     * @param args args.
+     */
+    public static void main(final String[] args) {
+        final Scanner scanner = new Scanner(System.in, "UTF-8");
+        Ocean ocean = new Ocean();
+        System.out.println("Welcome to BattleShip Game! \n");
+        ocean.placeAllShipsRandomly();
+
+        while (true) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)